packages/sync/src/constants.ts

```diff
@@ -24,3 +24,21 @@ export const FACT_LOOKUP_UP_TO_TICK = 'lastSuccessfulLookupTableUpdate';
 export const FACT_SYNC_TRIGGER_CONTROL = 'syncTrigger';
 
 export const COLUMNS_EXCLUDED_FROM_SYNC = ['updatedAtSyncTick'];
+
+export const NON_SYNCING_TABLES = [
```

Comment:
In another PR, you removed [this comment](https://github.com/beyondessential/tupaia/pull/6221/files/2399034cc7a5daa27b6467d3b8f4a20086bb1b6d..530f5b3abfe6d7122f7009e0231c7891d0e1fc71?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&show-viewed-files=true&show-deleted-files=false#diff-6421baf7a663650038c74c3f943fdbb27c56bb308ad686704a89e7a21b32ccffL2)

Do we need a new one referring to this const?